sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/orc/OrcQuerySuite.scala

```diff
@@ -830,6 +830,27 @@ abstract class OrcQuerySuite extends OrcQueryTest with SharedSparkSession {
       }
     }
   }
+
+  test("SPARK-37463: read/write Timestamp ntz or ltz to Orc uses UTC timestamp") {
+    TimeZone.setDefault(TimeZone.getTimeZone("America/Los_Angeles"))
+    sql("set spark.sql.session.timeZone = America/Los_Angeles")
```

Comment:
IIUC this orc bug is only about the the jvm timezone, not the spark session timezone, we don't need to set it.